Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/revocation/storage/RevocationRepository.kt

```diff
@@ -22,6 +25,11 @@ class RevocationRepository @Inject constructor(
             tag = TAG,
             scope = appScope
         )
+
+    fun clear() {
```

Comment:
Please add to DataResetModule? in order not to forget it

Reply:
good point!
adjusted.